src/main/java/uk/gov/hmcts/reform/preapi/model/Booking.java

```diff
@@ -22,4 +23,16 @@ public class Booking {
     private Timestamp createdAt;
     private Timestamp modifiedAt;
     private Set<Participant> participants;
+
+    public Booking(uk.gov.hmcts.reform.preapi.entities.Booking bookingEntity) {
```

Comment:
better to import this to make the function definition neater?

Reply:
it will clash with the model of the same name :( We could rename the classes to append the type so `BookingModel` and `BookingEntity` ?

Reply:
Will refactor in next PR